packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1662,7 +1652,6 @@ test.serial('writeGranulesFromMessage() removes preexisting granule file from Po
     granule,
     granulePgModel,
     knex,
-    providerCumulusId,
```

Comment:
Where did this change come from?

Reply:
It's a good change, but it looks like a refactor from the merge branch to eliminate a signature mismatch between the test and the call?     This seems fine, assuming it's not some other git ghost, although nit:there are other references in the file.

Reply:
`providerCumulusId` seems to be passed in in the feature branch. I did a manual removal of a lot of them, but I think some of them came back when I resolved the merge conflicts and not sure exactly why that is. It was taken out in main. Not sure if I should go back through the whole file again and manually take them out.